src/models/Manifest.js

```diff
@@ -1,14 +1,99 @@
-const { viewResultsFromKeys } = require("../resources/couch");
+const { NotFoundError } = require("../errors");
+const { getDocument, viewResultsFromKeys } = require("../resources/couch");
 const { multiTextValueToSingle } = require("../util");
 
 class Manifest {
   static #DB_NAME = "manifest";
   #id;
+  #slug;
+  #label;
+  #summary;
+  #ocrPdf;
+  #type;
+  #freezeParameters;
+  #canvases = [];
+  #parents = [];
 
   constructor(id) {
     this.#id = id;
   }
+  get id() {
+    this.#id;
+  }
+  async initialize() {
+    let document;
+    try {
+      document = await getDocument(Manifest.#DB_NAME, this.#id);
+    } catch (error) {
+      throw error.status === 404
+        ? new NotFoundError(`Manifest ${this.#id} not found.`)
+        : error;
+    }
+    this.#slug = document.slug;
+    this.#label = multiTextValueToSingle(document.label);
+    if (document.summary)
+      this.#summary = multiTextValueToSingle(document.summary);
+    this.#ocrPdf = document.ocrPdf;
+    this.#type = document.type;
+    this.#freezeParameters = document.freezeParameters;
+  }
+  //load Parents
+  #loadParents = async () => {
+    let rows;
+    try {
+      rows = await viewResultsFromKeys("Collection", "access", "items", [
```

Comment:
`"collection"` needs to be lowercase.

We have to refactor the loadParents method in Collection.js to a static method that accepts a noid, since both collection noids and manifest noids work.

Reply:
Yeah its in small, i havent pushed the changes from morning.